pypsa/variables.csv

```diff
@@ -4,6 +4,7 @@ Generator,status,False,False,True
 Generator,start_up,False,False,True
 Generator,shut_down,False,False,True
 Generator,p_nom,False,True,False
+Generator,n_opt,False,False,True
```

Comment:
We started from the Generator and as discussed other components will follow once agreed on the structure.
Do you like the "n_opt" name of the variable?

Reply:
we just discussed with fabian to name the variable as "n_mod"